build.gradle

```diff
@@ -31,10 +31,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.1'
-
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
```

Comment:
This auto-generated comment can be found in each build.gradle file. I don't think we need it anymore.